src/js/edu-benefits/5490/config/form.js

```diff
@@ -201,6 +198,9 @@ const formConfig = {
               previousBenefits: _.merge(
                 _.unset('properties.veteranFullName', previousBenefits),
                 {
+                  // Doesn't let us pass even if view:claimedSponsorService is false
+                  //  (which makes the ssn box is hidden)
+                  // required: ['veteranSocialSecurityNumber'],
```

Comment:
Would something like this work here?
https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/js/edu-benefits/1995/config/form.js#L71
I.e., only requiring it if `view:claimedSponsorService` is true